rfcs/text/0000-dss-deletion-daemon.md

```diff
@@ -0,0 +1,416 @@
+### DCP PR:
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# RFC Deletion of data in the DCP
+
+## Summary
+Establish a secure and controlled process for deleting data from the Data Store.
+
+
+## Author(s)
+
+ [Trent Smith](mailto:tsmith12@ucsc.edu)
+ 
+ [Hannes Schmidt](mailto:hannes@ucsc.edu)
+
+## Shepherd
+ [Rhian Anthony](mailto:ranthony@broadinstitute.org)
+
+## Motivation
+
+One of the DCP's primary objectives is to ensure the durability of the
+(meta)data it contains and to maintain a complete record of the changes made to
+it. The DCP achieves this by never overwriting a (meta)data file but instead
+treating an update to such a file as the addition of a new version of that
+file. Instead of physically removing a file version, the DCP hides that
+file version behind a deletion marker.
+
+Under certain circumstances however, the **physical deletion** of a file version
+is required. The first section of this specification describes the
+circumstances and mechanisms for logical and physical deletion of files and the
+bundles containing them. The second section details the changes to the API required to manage the deletion process.
+The final section details the process of deleting data from the DSS, and design details.
+
+### User Stories
+
+* As a user of the DSS, I would like to know what bundles have been deleted, so I can keep my personal index up-to-date. 
+* As a data wrangler of the DSS, I need to remove data from the DSS, because there was no consent from the donor to share the data.
```

Comment:
From a strategic viewpoint, this user story should be written from the perspective of the user informing us of the consent issues. The Wrangler is almost always only an messenger.

I would suggest

As a data contributor, I need to remove data from the DSS as I have realised the consent it was collected under does not allow it to be hosted by the DCP.